examples/post_training_quantization/openvino/yolov8/requirements.txt

```diff
@@ -0,0 +1,2 @@
+ultralytics==8.0.43
+openvino-dev==2023.0.0
```

Comment:
After removal of `onnx` as a requirement in the last commit, this test now fails for me with 

`ONNX: export failure ❌ 2.0s: No module named 'onnx'`

error. Could you please check again?

Reply:
done